Actions/AL-Go-Helper.ps1

```diff
@@ -103,6 +103,22 @@ function ConvertTo-HashTable() {
     $ht
 }
 
+function ConvertTo-JsonArray() {
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory=$true, ValueFromPipeline=$true)] 
+        $array
+    )
+    if ($array.Count -le 1) {
```

Comment:
"less or equal" or "equal" here?

Do we expect empty array to be serialized as "[]" or as ""?

Reply:
[]
as
'[]' | ConvertFrom-Json returns an empty array
and
'' | ConvertFrom-Json returns null

Reply:
I removed `ConvertTo-JsonArray`.

It turned out `ConvertTo-Json $array` does its jobs properly. `$array | ConvertTo-Json` unwraps the array, so there might be side-effects if there is a single value in the array.